docs/guide-events.md

```diff
@@ -0,0 +1,55 @@
+---
+id: guide-events
+title: Considerations for fireEvent
+---
+
+## Interactions vs. events
+
+Based on [the Guiding Principles](guiding-principles.md), your test should
+resemble how users interact with your code (component, page, etc.) as much as
+possible. With this in mind, you should be aware that `fireEvent` is not actually
+how the user interacts with your application.
```

Comment:
This is too strongly worded. In fact, none of what our tests are doing is how the user interacts with our app. We're just making trade-offs. Instead, you could say this:

```
With this in mind, you should know that `fireEvent` isn't _exactly_ how the user interacts with your application, but it's close enough for most scenarios.
```

Reply:
done